problems/[230619] Top K Frequent Words.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/top-k-frequent-words
+ *
+ * @param {string[]} words
+ * @param {number} k
+ * @return {string[]}
+ */
+const topKFrequent = function (words, k) {
+  const wordFrequent = {};
+
+  words.forEach((word) => {
+    wordFrequent[word] ? (wordFrequent[word] += 1) : (wordFrequent[word] = 1);
+  });
+
+  const wordFrequentArray = Object.entries(wordFrequent);
+
+  wordFrequentArray
+    .sort((a, b) => {
+      return b[1] - a[1];
+    })
+    .sort((a, b) => {
+      if (a[1] === b[1]) {
+        return a[0] > b[0] ? 1 : -1;
+      }
+    });
+
+  const answer = wordFrequentArray
+    .map((word) => {
+      return word[0];
+    })
+    .splice(0, k);
```

Comment:
slice도 배열을 반환하기 때문에 in place 메서드인 splice 말고 slice는 어떨까요? 

```suggestion
  const answer = wordFrequentArray.slice(0, k).map(word => word[0]);
```
slice로 먼저 자른 후 map을 사용하면 배열 순회를 좀 더 줄일 수 있을 것 같아요!